.travis.yml

```diff
@@ -2,28 +2,27 @@ language: php
 
 sudo: false
 
-php: [5.3, 5.4, 5.5, 5.6, 7.0, hhvm]
+php: [5.4, 5.5, 5.6, 7.0, 7.1, hhvm]
```

Comment:
Removing CI on 5.3 without dropping support is a no-go. This is the best way to drop support without telling it to Composer.

Reply:
There is a trick to force it to keep running on 5.3 precise for now.

Reply:
What do you prefer @stof? Using the trick for Travis (I know which one) or dropping the support?

Reply:
To fix PHP 5.3 build you need to move it from general config to matrix, that uses `precise` OS version for that PHP version specifically.

Reply:
@aik099 yes I know ;).

Reply:
@dunglas , not fixed.

Reply:
please add PHP 7.2 in this list.

And I would remove HHVM, as they decided to not care about PHP parity anymore.